src/commands/periscope/periscope.ts

```diff
@@ -0,0 +1,248 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as buffer from 'buffer';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { parseResource } from '../../azure-api-utils';
+import * as tmpfile from '../utils/tempfile';
+import { getClusterKubeconfig } from '../../extension';
+import { getExtensionPath, longRunning } from '../utils/host';
+import {
+  getClusterDiagnosticSettings,
```

Comment:
You're indenting by 4 elsewhere.  Is this a different indent standard or are you using tabs instead of spaces?  Use editorconfig or something to keep things consistent.